### DISCUSSION TITLE
[Question] strategy for MAUI macOS

### DISCUSSION THREAD

**Discussion by nor0x** [dotnet/maui#235] [General]
when .NET MAUI was announced the best news for me personally was to hear that macOS support is coming directly from Microsoft. I'm in a team that uses Xamarin.Forms on macOS quite heavily and the current experience is not good to be honest. Issues like https://github.com/xamarin/Xamarin.Forms/issues/10101, https://github.com/xamarin/Xamarin.Forms/issues/10789, https://github.com/xamarin/Xamarin.Forms/issues/10936, https://github.com/xamarin/Xamarin.Forms/issues/11694, https://github.com/xamarin/Xamarin.Forms/issues/10947, https://github.com/xamarin/Xamarin.Forms/issues/10918 are huge showstoppers for us. I know that currently macOS is still community backed in Xamarin.Forms but i think basic UI functionality like coloring a labels' text should work out of the box.

Basically my question is if there is already a timeframe or a strategy for existing XF issues related to macOS, most the items are open for months and get pushed back every sprint. So am i right to assume that issues like the ones mentioned above wouldn't make it into the first preview of MAUI which is expected to arrive this year.

I really look forward to MAUI and will continue to work on info, repros and even PRs for Xamarin.Forms issues. I just hope to have a stable, future-proof UI framework for macOS that our software can rely on.

**Comment by davidortinau** [comment]
On our [roadmap](https://github.com/dotnet/maui/wiki/Roadmap) we don't plan to start the desktop portion of the work until early 2021. Those issues are others may be address beforehand, but certainly our attention turns there when we get further into the .NET MAUI timeline.

**Reply by jamesmcmenamin** [reply]
Just adding in that a number of these items are really simple to sidestep until the official fix's area available via CR's. I know personally I want to find a way to fix things / do things without using CR's as much as possible because it feels like a failure if I have to CR but Xamarin (and I hope continues in MAUI) move at an amazing clip, and you can always fix the issue and put it up on PR.
